### PR TITLE
One more unit test

### PR DESCRIPTION
**Pull request by Bengt-M**

### CODE REVIEW

File: src/test/unit/rc_controls_unittest.cc [hunk 1]
```diff
@@ -235,10 +235,10 @@ class RcControlsAdjustmentsTest : public ::testing::Test {
             .thrExpo8 = 0,
             .rates = {0, 0, 0},
             .dynThrPID = 0,
-            .tpa_yaw_rate = 0,
             .rcYawExpo8 = 0,
             .tpa_breakpoint = 0,
-            .tpa_yaw_breakpoint = 0
+            .tri_dynamic_yaw_minthrottle = 1000,
```
Comment (lkaino) [inline src/test/unit/rc_controls_unittest.cc]: These are in percents, so nominal values would be  around 100.

**Comment by lkaino** [comment]
On what environment do you run the test cases? They run fine on a native Linux but on Cygwin I'm getting build errors:
- cc is not defined
- pthread not found

I can make them run by defining the cc and removing the -pthread. Do you know a proper way to get them compiling on Cygwin?

**Comment by pqueiros** [comment]
Just to understand a bit better the problem, the you have that issue with all unit tests right?
by the way I use Linux. Smells like library locations are different in cygwin.

**Comment by lkaino** [comment]
Yes it's the same thing with compiling all of the tests and individual .cc files.

**Comment by Bengt-M** [comment]
I have a Win-PC and I use Eclipse. Behind I have cygwin64 and GNU Tools ARM
Embedded 5.3

When I installed cygwin I had to make sure I got the C++/G++ parts as well,
not just C. That may be your problem.

In excel (in the global settings) I have environment vars to point to these
two tool sets.
Window :: Preferences :: C/C++/Build/Environment/PATH
This is global for Excel so I don't have to fix it for every project I
import.

I build using the supplied makefile, using the "make target" view in
Excel. Works very well.

Another possible source of problems:
One time when I did the routine Import :: Existing Code as Makefile Project
I did not have the C++ check-box ticked.
The tests need C++. I could never understand how to fix it so I
just imported again. And then it worked.

I don't have the PATH to the tool sets in Windows anymore but I never need
to build outside of Eclipse.
Only reason would be to run that script MakeAll.sh.

2016-05-19 12:53 GMT+02:00 lkaino notifications@github.com:

**Comment by Bengt-M** [comment]
Ok, now at the PC again.
When I read your description again it sounds like maybe you are running without any gui like Excel.
To get that working is even more easy.
Reinstall cygwin and make sure you add the c++/g++ stuff.
Put both cygwin and gnu arm in system PATH
That's it. Just tried it.

making the tests is best done with 'make -k test' so it doesn't stop at first error.

**Comment by lkaino** [comment]
Thanks, re-install of Cygwin did it.

When you're talking about Excel do you mean Eclipse? I'm using Eclipse for development + compiling.

**Comment by Bengt-M** [comment]
Eclipse it is :)
Too much Excel at work now...

2016-05-20 12:45 GMT+02:00 lkaino notifications@github.com:

**Pull request merged**